### PR TITLE
Feature/teachers pixel

### LINKED ISSUE #2386: Add tracking pixel to "Get the courses" button
We'd like to know how many people are clicking on the "Get the courses" button at https://amp.dev/documentation/courses/teachers.

To do this, I'm planning to create a new event in GA. I'd then like to give that "Get the courses" button an ID and track it with a trigger as described [here](https://developers.google.com/analytics/devguides/collection/amp-analytics/).

I'm looking at the `analytics.j2` partial. I guess I'd need to add something to the podspec to let the config include triggers. At this point, I thought I should ask others for guidance!

/cc @matthiasrohmer @sebastianbenz @pbakaus @CrystalOnScript

### PR DESCRIPTION
**Pull request by morsssss**
Resolves #2386 .

I decided to use this method to check which page is in use... let me know if another method is preferred.

### CODE REVIEW

File: frontend/templates/views/partials/analytics.j2 [hunk 1]
```diff
@@ -9,6 +9,20 @@
         }
       }
     }
+{% if '/courses/' in doc.pod_path and doc.title == 'Teachers' %}
```
Comment (sebastianbenz) [inline frontend/templates/views/partials/analytics.j2]: Will this work for translated versions? Why not only use the path?

Comment (matthiasrohmer) [reply frontend/templates/views/partials/analytics.j2]: Sebastian was a little bit quicker 😁 Check for `doc.pod_path == '/content/amp-dev/documentation/courses/teachers.md'` instead.

File: frontend/templates/views/partials/analytics.j2 [hunk 2]
```diff
@@ -9,6 +9,20 @@
...
+    ,
```
Comment (matthiasrohmer) [inline frontend/templates/views/partials/analytics.j2]: Does not needs to be done in this PR: we should refactor the analytics configuration to be a Python dict in the template which is then printed with `config|tojson` - that's a little less verbose.

**Comment by morsssss** [comment]
Yeah, ok, well, that's the other way I was considering. I see now that it's better :)

To match your phenomenally fast reviews, I've rapidly changed this, checked it, and resubmitted.

**Pull request merged**